microcosm_flask/conventions/landing.py

```diff
@@ -85,7 +85,7 @@ def render_landing_page():
         """
         config = graph.config_convention.to_dict()
         env = get_env_file_commands(config, graph.metadata.name)
-        health = graph.health_convention.to_dict()
+        health = graph.health_convention.to_dict(full=True)
```

Comment:
Is this needed? You are hardcoding `full=True` in the template so doesn't look like this addition will be used during the render?

Reply:
The template change below only updates a link to the health endpoint. I thought it was helpful to update that because it might help people remember the `full` param exists. I doubt anyone knows that this link is even there.

The rendering of the health info happens here server side. This is the real change.